File: Server/Application/Hubs/Models/TurnVillianEvent.cs
﻿using Server.Game.Models.Match;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Server.Application.Hubs.Models
{
    public class TurnVillianEvent : IngameEvent
    {
        public List<TicketBagForPlayer> DetectiveTickets { get; set; }
        public List<ChoosableRoute> Routes { get; set; }
    }
}

File: Server/Client/src/app/models/bindings/turn-villian-event.ts
import { TicketBag } from "../ticket-bag";
import { ChoosableRoute } from "./choosable-route";
import { IngameEvent } from "./ingame-event";
import { TicketBagForPlayer } from "./ticket-bag-for-player";

export interface TurnVillianEvent extends IngameEvent {
  detectiveTickets: TicketBagForPlayer[];
  routes: ChoosableRoute[];
}
